lib/src/asymmetric_operator.dart

```diff
@@ -132,26 +133,25 @@ class _AsymmetricEncrypter extends Encrypter<Key> with _AsymmetricOperator {
   _AsymmetricEncrypter(Identifier algorithm, Key key) : super._(algorithm, key);
 
   @override
-  pc.AsymmetricBlockCipher get _algorithm => super._algorithm;
+  pc.AsymmetricBlockCipher get _algorithm =>
+      super._algorithm as pc.AsymmetricBlockCipher;
 
   @override
   Uint8List decrypt(EncryptionResult input) {
-    _algorithm.init(
-        false,
-        pc.ParametersWithRandom(keyParameter, null //pc.SecureRandom('Fortuna')
```

Comment:
@jpeiffer @rbellens curious if you know the right way to replace this null now that we can't pass null to PC.